src/components/Film/Film.js

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import { Card } from "react-bootstrap";
+import { NavLink } from "react-router-dom";
+import { ABOUTFILM_ROUTE } from "../../utils/consts";
+import { useDispatch } from "react-redux";
+import { changeFilm } from "../../store/FilmReducer";
+
+const Film = (props) => {
+  const dispatch = useDispatch();
+
+  const changeFilmId = (filmId) => {
+    dispatch(changeFilm(filmId));
+  };
+
+  return (
+    <Card style={{ width: "18rem", margin: "5px" }}>
+      <Card.Img variant="top" src={props.src} />
+      <Card.Body>
+        <Card.Title>{props.title}</Card.Title>
+        <Card.Text>{props.desc}</Card.Text>
+        <NavLink
+          onClick={() => changeFilmId(props.id)}
```

Comment:
First of all, take a look at useCallback. Secondly, you don't need changeFilId func, you just may use () => dispatch(changeFil(filmId)).

Reply:
Fixed